frontend/src/containers/Home.css

```diff
@@ -1,3 +1,22 @@
 .usa-search__submit-text {
   vertical-align: text-top !important;
 }
+
+.cards ul {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 0;
+}
+
+.cards ul li {
+  list-style: none;
+  flex: 0 0 33.333333%;
+}
+
+.divbackground {
+  background-color: rgb(220, 220, 220);
+}
+
+.divborder {
+  border: 1px solid;
```

Comment:
there a border directive in USWDS too.

Why don't we use the USWDS card component instead of creating a custom one?

https://designsystem.digital.gov/components/card/

Reply:
If you check the code in our card component, it is identical to the USWDS code. Having our own card component allows further customization and extensions such as adding a link to the card title like what I am doing.

Reply:
If you check the code in our card component, it is identical to the USWDS code. Having our own card component allows further customization and extensions such as adding a link to the card title like what I am doing.

Reply:
Used the border class.